interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Save Reports
+author: Mary Sammal
+---
+# Save Reports
+Create a report's _backup copy_ to guarantee that you can revert the report to its original state further.
```

Comment:
"Create a report's _backup copy_ to guarantee that you can revert the report to its original state further" What? This sentence does not make any sense. 'guarantee'? 'further'? Rewrite.

Reply:
I decided to simplify the information:

Use one of the following commands.

Reply:
You can use one of the following commands to...: